### PR TITLE
add doc for `tidb_txn_entry_size_limit`

### PR DESCRIPTION
**Pull request by zyguan**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add document for `tidb_txn_entry_size_limit`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/49237

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -4532,6 +4532,20 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 >
 > TiDB Dashboard 中的 Top SQL 页面目前只显示消耗负载最多的 5 类 SQL 查询，这与 `tidb_top_sql_max_time_series_count` 的配置无关。
 
+### `tidb_txn_entry_size_limit` <span class="version-mark">从 v7.6.0 版本开始引入</span>
+
+- 作用域: GLOBAL | SESSION
```
Comment (Oreoxmt) [inline system-variables.md:4537]:
```suggestion
- 作用域：GLOBAL | SESSION
```

File: system-variables.md [hunk 2]
```diff
@@ -4532,6 +4532,20 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`0`
+- 范围： `[0, 125829120]`
```
Comment (Oreoxmt) [inline system-variables.md:4542]:
```suggestion
- 范围：`[0, 125829120]`
- 单位：字节
```

File: system-variables.md [hunk 3]
```diff
@@ -4532,6 +4532,20 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+- 这个变量用于动态修改 [`performance.txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-从-v50-版本开始引入) 配置项，其含义与对应配置项相同。变量默认值为 0 ，表示默认使用配置项的值，当设置为非 0 值时，则优先使用该值作为 `txn-entry-size-limit` 的值。
```
Comment (Oreoxmt) [inline system-variables.md:4543]:
```suggestion
- 这个变量用于动态修改 TiDB 配置项 [`performance.txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-从-v50-版本开始引入)，其含义与对应配置项相同，即用于限制 TiDB 单行数据的大小。该变量默认值为 `0`，表示默认使用配置项的值。当设置为非 `0` 值时，优先使用该变量的值作为 `txn-entry-size-limit` 的值。
```

File: system-variables.md [hunk 4]
```diff
@@ -4532,6 +4532,20 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+
+> **注意：**
+>
+> 修改 SESSION 作用域的变量时，只会影响当前用户会话，TiDB 内部会话并不受影响。这可能造成 TiDB 内部事务的 entry size 超过配置项的限制，从而导致事务失败。因此当需要在线调高限制时，我们建议优先修改 GLOBAL 作用域的变量。
```
Comment (Oreoxmt) [inline system-variables.md:4547]:
```suggestion
> 修改 SESSION 作用域的变量仅影响当前用户会话，不会影响 TiDB 的内部会话。这可能导致 TiDB 内部事务的单行数据大小超过限制，从而导致事务失败。如需在线调高限制，建议优先修改 GLOBAL 作用域的变量。
```

Comment (Oreoxmt) [reply system-variables.md:4547]: 建议在配置项文档中添加 tidb_txn_entry_size_limit 系统变量的使用场景和说明，例如 L482：
```
+ 从 v7.6.0 开始，你可以使用 [`tidb_txn_entry_size_limit`](/system-variables.md#tidb_txn_entry_size_limit-从-v760-版本开始引入) 系统变量动态修改该配置项的值。
```
 
https://github.com/pingcap/docs-cn/blob/b333f5f5bff171e56994e9f5e692ce2028298e2f/tidb-configuration-file.md?plain=1#L476-L482

**Review by Oreoxmt — APPROVED** [review]

File: system-variables.md [hunk 5]
```diff
@@ -4745,6 +4745,21 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`0`
+- 范围：`[0, 125829120]`
+- 单位：字节
+- 这个变量用于动态修改 TiDB 配置项 [`performance.txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-从-v50-版本开始引入)，其含义与对应配置项相同，即用于限制 TiDB 单行数据的大小。该变量默认值为 `0`，表示默认使用配置项的值。当设置为非 `0` 值时，优先使用该变量的值作为 `txn-entry-size-limit` 的值。
+
+> **注意：**
+>
+> 修改 SESSION 作用域的变量时，只会影响当前用户会话，TiDB 内部会话并不受影响。这可能造成 TiDB 内部事务的 entry size 超过配置项的限制，从而导致事务失败。因此当需要在线调高限制时，我们建议优先修改 GLOBAL 作用域的变量。
```
Comment (Oreoxmt) [inline system-variables.md:4761]:
```suggestion
> 修改 SESSION 作用域的变量仅影响当前用户会话，不会影响 TiDB 的内部会话。这可能导致 TiDB 内部事务的单行数据大小超过配置项的限制，从而导致事务失败。如需在线调高限制，建议优先修改 GLOBAL 作用域的变量。
```

**Review by cfzjywxk — APPROVED** [review]
LGTM

File: system-variables.md [hunk 6]
```diff
@@ -4745,6 +4745,21 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`0`
+- 范围：`[0, 125829120]`
+- 单位：字节
+- 这个变量用于动态修改 TiDB 配置项 [`performance.txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-从-v50-版本开始引入)，其含义与对应配置项相同，即用于限制 TiDB 单行数据的大小。该变量默认值为 `0`，表示默认使用配置项的值。当设置为非 `0` 值时，优先使用该变量的值作为 `txn-entry-size-limit` 的值。
+
+> **注意：**
+>
+> 修改 SESSION 作用域的变量仅影响当前用户会话，不会影响 TiDB 的内部会话。这可能导致 TiDB 内部事务的单行数据大小超过配置项的限制，从而导致事务失败。如需在线调高限制，建议优先修改 GLOBAL 作用域的变量。
```
Comment (qiancai) [inline system-variables.md:4761]:
```suggestion
> 使用 SESSION 作用域修改该变量时仅影响当前用户会话，不会影响 TiDB 的内部会话。这可能导致 TiDB 内部事务的单行数据大小超过配置项的限制，从而导致事务失败。如需在线调高限制，建议优先使用 GLOBAL 作用域修改该变量。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**